index.html

```diff
@@ -234,10 +234,10 @@ <h1>03</h1>
 								<div class="desc">
 									<h4>Custom Desktop Environment</h4>
 									<p>
-										Archcraft has no desktop environment. it's uses Openbox and Bspwm, both are window managers. Each window manager is highly configured, gives you a feel like a whole desktop environment. There are multiple styles for each WM, which change everything (BG, Panel, Menus, Theme & icons, Fonts Etc) with just one click. It's totally on you which styles you prefer.
+										Archcraft has no desktop environment. It uses Openbox and Bspwm which are are both window managers. Each window manager is highly configured, giving you the feel of a complete desktop environment. There are multiple styles for each WM, which change everything (BG, Panel, Menus, Theme & icons, Fonts, etc.) with just one click. It's totally on you which styles you prefer.
```

Comment:
are are

Reply:
‘BG’… is that wallpaper, @adi1090x?

Reply:
I am pretty sure yes, so I am going to go ahead and expand preemptively it to wallpaper

Reply:
Yeah, it's wallpaper (background)